include/util/Base64.h

```diff
@@ -0,0 +1,67 @@
+/**
+ *
+ * @copyright Copyright (c) 2022 ApertureData Inc.
+ *
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <cassert>
+#include <openssl/evp.h>
+
+class Base64 {
+public:
+    // static methods only
+    Base64() = delete;
+
+    static constexpr std::size_t encoded_bytes( std::size_t bytes )
```

Comment:
encoded_bytes and decoded_bytes are inverse functions of each other. Can we have a simple test to check that.

Reply:
They're not quite inverse in that `encoded_bytes` always returns a multiple of 4 and `decoded_bytes` a multiple of 3.  So `decoded_bytes( encoded_bytes( 5 ) ) == 6`.

But you make a good point about adding a test.

Reply:
Of course you are correct.

It is strange that `decoded_bytes( encoded_bytes( 5 ) ) == 6`.